### ISSUE TITLE
BitmapData.draw() doesn't work properly with mask in HTML5

### ISSUE THREAD

**Issue by guruas3** [openfl/openfl#1769]
New stencil buffer mask works well for me but seems like it's not taken into account during BitmapData.draw call.

Here is a simple example:
```haxe
//addChild(holder);
	
var rect:Sprite = new Sprite();
rect.graphics.beginFill(0x000000);
rect.graphics.drawRect(0, 0, 400, 300);
rect.graphics.endFill();

var circle:Sprite = new Sprite();
circle.graphics.beginFill(0x00ffcc);
circle.graphics.drawCircle(0, 0, 100);
circle.graphics.endFill();

circle.x = 150;
circle.y = 150;
rect.mask = circle;
holder.addChild(rect);
holder.addChild(circle);

bitmapData = new BitmapData(Std.int(holder.width), Std.int(holder.height));
bitmapData.draw(holder);

var bmp:Bitmap = new Bitmap(bitmapData);
addChild(bmp);
```

Flash output:
![2017-11-23_1702](https://user-images.githubusercontent.com/981231/33181328-234e5c80-d070-11e7-853e-afe4ba046fab.png)

HTML5 output:
![2017-11-23_1702_001](https://user-images.githubusercontent.com/981231/33181331-26bf6fb2-d070-11e7-80a9-19bc132c9c31.png)

Using latest versions of openfl 6.5.0 and lime 5.8.2
[BitmapDataIssue.zip](https://github.com/openfl/openfl/files/1499751/BitmapDataIssue.zip)

**Comment by NenadBojkovski** [comment]
Hi does this maybe fixes your issue ?
https://github.com/openfl/openfl/pull/1770

**Comment by guruas3** [comment]
@NenadBojkovski no it doesn't fix it. i've just installed the latest version of openfl with your pull request merged and i have the same problem with html5.

**Comment by guruas3** [comment]
I've noticed that during the `bitmapData.draw` some graphics commands are missing from `renderMask()` in CanvasGraphics.hx
https://github.com/openfl/openfl/blob/656a5c2352ab40444ebddf422ff73651f00ed440/src/openfl/_internal/renderer/canvas/CanvasGraphics.hx#L1462
Maybe this could be the problem.

**Comment by mastef** [comment]
With the solution from #1831 I'm getting this result : 

![screenshot 2018-02-26 15 59 48](https://user-images.githubusercontent.com/762297/36661553-1f6410ae-1b0e-11e8-837c-5865ee3c74d5.png)

**Comment by guruas3** [comment]
Joshua has fixed the issue in 8.0.1 release
